sdk/applicationinsights/Azure.ResourceManager.ApplicationInsights/src/Customized/ApplicationInsightsComponentCollection.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Resources;
+
+namespace Azure.ResourceManager.ApplicationInsights
```

Comment:
Since this RP hasn't been GAed yet, we don't need to fix breaking changes by using customization code.

Reply:
removed them